Lib/fontTools/feaLib/builder.py

```diff
@@ -8,15 +8,19 @@
 import itertools
 
 
-def addOpenTypeFeatures(featurefile_path, font):
-    builder = Builder(featurefile_path, font)
+def addOpenTypeFeatures(font, featurefile_path=None, featurefile_data=None):
```

Comment:
Ok, i'll change that.

Adrien, I'm also wondering, why did you choose to pack (fea_path, fea_data) into a tuple and pass it down to Builder and Parser as a tuple until the Lexer unpacks it, instead of as two separate arguments?